app/models/notification.rb

```diff
@@ -5,6 +5,7 @@ class Notification < ApplicationRecord
   validates :content, presence: true
 
   delegate :convict_phone, :convict, to: :appointment
+  delegate :id, to: :appointment, prefix: true
```

Comment:
je suis pas clair sur l'interet de la chose

Reply:
Le point 1 --> https://acuments.com/10-rails-antipatterns-to-avoid-writing-well-reasoned-code.html